src/Map.jsx

```diff
@@ -108,33 +110,42 @@ class Map extends Component {
    * return coordinates in an array [lat, lon]
    */
   getCoordinateArray(stop) {
-    return [stop.lat, stop.lon];
+    console.log(this);
+    return [stop.lon, stop.lat];
```

Comment:
Would this be cleaner if it were done in-line? `(stop) => ([stop.lon, stop.lat])`? Or even better, making `stop` a class and calling `stop.getCoordinateArray()` :)